sympy/geometry/line.py

```diff
@@ -897,18 +1006,18 @@ def projection(self, other):
 
         other : Point or LinearEntity (Line, Ray, Segment)
 
-        Returns
-        =======
-
-        projection : Point or LinearEntity (Line, Ray, Segment)
-            The return type matches the type of the parameter ``other``.
-
         Raises
         ======
 
         GeometryError
             When method is unable to perform projection.
 
+        Returns
+        =======
+
+        Point or LinearEntity (Line, Ray, Segment) :
+            Projection of ``other`` on this linear entity.
```

Comment:
The message should be about what other pertains to, not about 'other'

Reply:
Should I make it like "returns the projection of ``other`` on this linear entity." ?